setup.py

```diff
@@ -12,6 +12,9 @@
     url='https://github.com/imsweb/django-history-triggers',
     license='BSD',
     packages=find_packages(),
+    install_requires=[
+        six>=1.12,
```

Comment:
Is there something about version 1.12 of `six` you're using? If not, I would drop the version requirement, since I think `string_types` has been there all along.

Reply:
Also, does this even work? Shouldn't it be quoted?